packages/dev-server-hmr/src/hmrClientScript.ts

```diff
@@ -44,7 +44,7 @@ export class HotModule {
 
     sendMessage({ type: 'hmr:accept', id: this.id });
     this[moduleState] = HmrState.Accepted;
-    this[acceptHandlers].add(callback ?? () => {});
+    this[acceptHandlers].add(callback || function() {});
```

Comment:
i'd still rather we use the null coalesce operator here, its just missing parentheses:

```ts
add(callback ?? (() => {}));
```